test/system/map_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+require "application_system_test_case"
+# https://guides.rubyonrails.org/testing.html#implementing-a-system-test
+
+class MapTest < ApplicationSystemTestCase
+  Capybara.default_max_wait_time = 60
```

Comment:
Hey @nstjean, increasing the wait time seems to do the trick.. I changed this to` Capybara.default_max_wait_time = 120`

Reply:
actually 80 works too

Reply:
Ok I'm trying with 120 just to be safe. ;) If that works I'll change it to 80.